Dokan/Base/TargetType.swift

```diff
@@ -0,0 +1,33 @@
+//
+//  TargetType.swift
+//  Dokan
+//
+//  Created by Clara Mounir Adly on 22/06/2022.
+//
+
+import Foundation
+import Alamofire
+
+enum HTTPMethod: String {
+    case get = "GET"
+    case post = "POST"
+    case put = "PUT"
+    case delete = "DELETE"
+}
+enum Task {
+    case requestPlain
+    case requestParameters(parameters: [String: Any], encoding: ParameterEncoding)
+}
+protocol TargetType {
+    var baseURL: String {get}
+    var path: String {get}
+    var method: HTTPMethod {get}
+    var task: Task {get}
+    var headers: [String: String]? {get}
+}
+
+
+
+
+
+
```

Comment:
```suggestion
enum HTTPMethod: String {
    case get = "GET"
    case post = "POST"
    case put = "PUT"
    case delete = "DELETE"
}

enum Task {
    case requestPlain
    case requestParameters(parameters: [String: Any], encoding: ParameterEncoding)
}

protocol TargetType {
    var baseURL: String {get}
    var path: String {get}
    var method: HTTPMethod {get}
    var task: Task {get}
    var headers: [String: String]? {get}
}
```
Spacing and indentation